src/Meilisearch/ISearchable.cs

```diff
@@ -43,6 +43,6 @@ public interface ISearchable<T>
         /// Returns the numeric min and max values per facet of the hits returned by the search query.
         /// </summary>
         [JsonPropertyName("facetStats")]
-        IReadOnlyDictionary<string, IReadOnlyDictionary<string, float>> FacetStats { get; }
+        IReadOnlyDictionary<string, FacetStats> FacetStats { get; }
```

Comment:
There is a conflict as you are using both times `facetStats`. Is this an issue? Otherwise you can name the class `facetStat` and the other `facetStats`

Reply:
No is not a problem but you right actually is just one facet stat so it should be named `FacetStat`